src/client/menu/menu.c

```diff
@@ -5387,7 +5387,7 @@ PlayerDirectoryList(void)
 
         YQ2_COM_CHECK_OOM(s, "malloc()", MAX_QPATH * sizeof(char))
         
-        strncpy(s, t, MAX_QPATH - 1);         // MAX_QPATH - 1, gcc warns about truncation
+        Q_strlcpy(s, t, MAX_QPATH);
         s[strlen(s)] = 0;
```

Comment:
wonder, if that s the case, are all of those still needed ?

Reply:
I'm not sure what you mean. Are you talking about doing a full replace from strncpy to Q_strlcpy in the entire project?
If that's the case, there's only one way to find out... but not for this PR. I was going for a particular bug in this.

Reply:
no I meant the line I pointed out.

Reply:
he probably means the `s[strlen(s)] = 0;` line

which to be honest has always been pointless, if you think about what strlen() does

Reply:
exactly

Reply:
Sorry about the misunderstanding. Indeed, they are redundant now; they have been deleted.

Reply:
Yup, those null char assignments are needless. Even if n == 1, Q_strlcpy would use that 1 byte for the null char.

There is also something to be said about that kind of code being bad practice. If you are doing the same sequence over and over again, that's a hint you should wrap it into a function, otherwise it's only a matter of time before you mess up the sequence and cause bugs or crashes.